Docs: Write the Quick start CDN guide. (#15818)
ckeditor/ckeditor5: docs/getting-started/quick-start.md

Now you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately.

> **Importing and registering UI translations is optional for the English language.** For any other language, use imported translations, as in the {@link getting-started/setup/configuration setup section}.

Reviewer: ```suggestion
**Importing and registering UI translations is optional for the American English.** To use the editor in any other language, import translations as shown in the {@link getting-started/setup/configuration setup section}.
```

Reviewer 2: Unless we have separate translations for American and British (if so, we should reflect it in the docs immediately), this change makes very little sense.

Reviewer: As far as I can see, we have `en` (default), `en-au` and `en-gb`. That's why I suggested explicitly saying that **American English** is the default.

Now you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately.

> **Importing and registering UI translations is optional for American English.** To use the editor in any other language, use imported translations, as shown in the {@link getting-started/setup/configuration setup section}.

Reviewer 2: ```suggestion
**Importing and registering UI translations is optional for English.** To use the editor in any other language, use imported translations, as shown in the {@link getting-started/setup/configuration setup section}.
```

Reviewer 2: I insist. We do not have separate translations, do we?

Author: It seems that we have:

> As far as I can see, we have en (default), en-au and en-gb. That's why I suggested explicitly saying that American English is the default.

I also asked @pszczesniak, and he confirmed.

Reviewer 2: Why don't we have this documented and only list English in out langauge guide? o\_O This is a pretty meaningful rewrite.  
  
OK\< Keep American English, then. I'll split the issue on that.

Author: I looked through the language guides, and you're right - there is only "English" without dialects.

Reviewer 2: We actually do have community translations for these two bad boys: \`au\` and `gb` https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-core/lang/translations:

import 'ckeditor5/dist/styles.css';

await ClassicEditor
	.create( document.querySelector( '#editor' ), {
		plugins: [ Essentials, FormatPainter, SlashCommand ],
		toolbar: [ 'formatPainter' ],
		licenseKey: 'your-license-key'
	} )
```

## Using CKEditor&nbsp;5 from CDN

> CDN is an effortless method of running CKEditor 5. You can start using it in just a few steps and with a few tags.

Reviewer: I'd be cautious about using the word `effortless`, as - depending on the setup - it may require more work than the NPM setup. I'd suggest replacing `effortless` with `alternative`.

> CDN is an effortless method of running CKEditor 5. You can start using it in just a few steps and with a few tags.

Reviewer: I'd be cautious about using the word `effortless`, as - depending on the setup - it may require more work than the NPM setup. I'd suggest replacing `effortless` with `alternative`.

```suggestion
CDN is an alternative method of running CKEditor 5. You can start using it in just a few steps.
```

Author: Yeah, you're right. That was marketing jabber foremost. Will do.

### Obtain a license key

> To activate CKEditor&nbsp;5 premium features, you will need a commercial license. If you wish to purchase a commercial CKEditor&nbsp;5 license or a license to one of the premium features, [contact us](https://ckeditor.com/contact/?sales=true#contact-form) to receive an offer tailored to your needs. To obtain an activation key, please follow the {@link support/license-key-and-activation License key and activation} giuide.

Reviewer: ```suggestion
To activate CKEditor&nbsp;5 premium features, you will need a commercial license. If you wish to purchase a commercial CKEditor&nbsp;5 license or a license to one of the premium features, [contact us](https://ckeditor.com/contact/?sales=true#contact-form) to receive an offer tailored to your needs. To obtain an activation key, please follow the {@link support/license-key-and-activation License key and activation} guide.
```

<!-- Not sure we should duplicate this in QS: TODO: Describe the steps to obtain a license key, attaching some screenshots. -->

### CDN setup

> Start by attaching a link to the style sheet. It contains all styles for the editor's UI and content. You can also include your styles if you like. Refer to the content styles guide for more information.

Reviewer: Add link to the `content styles` guide.

Author: Sure, but we don't have this guide in the new structure yet.

```html
<link rel="stylesheet" href="<CDN_LINK>/ckeditor5/dist/styles.css" />
```

> Then, you need to attach the script with the JavaScript code. To simplify imports, you can use the feature available in browsers &ndash; the [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap). It maps the text used as the module specifier and the value that will replace the text when resolving the specifier. This way, import statements do not need to contain the full CDN URL.

Reviewer: This feature is not just used to make imports look nicer, so I would rephrase that so as not to tempt anyone to skip using it.

```suggestion
Next, we need to add an [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap) that allows us to map an easy-to-remember specifier (like `ckeditor5`) to the full URL of the file from the CDN. We use this browser feature to share an editor engine code between plugins.
```

Reviewer 2: I'd suggest sticking to "you" as this is the dominant across the whole guide.

Author: Good point. But I would leave "we" in the last sentence because there is a rationale for why we recommend this feature.

Reviewer 2: Sure.

```html
<script type="importmap">
	{
		"imports": {
			"ckeditor5": "<CDN_LINK>/ckeditor5/dist/index.min.js",

> 			"ckeditor5/": "<CDN_LINK>/ckeditor5/"

Reviewer 3: @filipsobol is this always needed?

Reviewer: It's only needed when importings translations, but I'm fine with having this here.

		}
	}
</script>
```

> After you  set up the import map, you can add plugins. In the following script tag, import the desired plugins and add them to the plugins array. Notice that both script tags have the corresponding `type` values.

Reviewer: ```suggestion
Once you've added the import map, you can access editor and its plugins using the `ckeditor5` specifier. In the following script tag, we import the desired plugins and add them to the `plugin' array. Note that this and the previous script tags have the appropriate `type` values.
```

Reviewer 2: No contractions.
